docs/stdlib/set.rst

```diff
@@ -159,16 +161,16 @@ Sets
 
     :index: if else ifelse elif ternary
 
-    Conditionally provide one or the other result.
+    Conditionally determines a logical action:
```

Comment:
```suggestion
    Produces one of two possible results based on the outcome of a condition.
```
I believe the original is more accurate in this context. Recommend rewriting in a way that better captures that original meaning.